### ISSUE TITLE
With /oauth/token request, the refresh_token is not Sanitized from request body

### ISSUE THREAD

**Issue by cdlliuy** [cloudfoundry/cli#1535]
when working on a cf cli plugin development work, I  tried to get AccessToken() from cf connections, in which a "/oauth/token" request is issued. 

With CF_TRACE=true enabled,  I found the refresh_token in request body is not Sanitized. 
As below: 
```
REQUEST: [2018-12-24T16:14:23+08:00]
POST /oauth/token HTTP/1.1
Host: xxxxxxx
Accept: application/json
Authorization: [PRIVATE DATA HIDDEN]
Content-Type: application/x-www-form-urlencoded
User-Agent: go-cli 6.41.0+dd4c76cdd.2018-11-28 / darwin

grant_type=refresh_token&refresh_token=eyJhbGciOiJIUzI1NiIsImtpZCI6ImRlZmF1bHQiLCJ0eXAiOiJKV1QifQ.eyJqdGkiOiI2NDVlMjhjOTIxNWY0OWJlOWZmMGEwMmQwNzk0ZjIxMi1y........&scope=
```

Checking  with Sanitize func here,  I noticed only "token" in json format is considered.   
https://github.com/cloudfoundry/cli/blob/master/cf/trace/trace.go#L21-L22

Is that possible to hide anything related to "token" from plain text as well?

**Comment by Ema93sh** [comment]
Hi @cdlliuy thanks for creating this issue. Yes, It is a bug in our legacy implementation. 

The refactored commands redacts the refresh tokens. However the plugins still uses our legacy implementation. This issue will be fixed once we change our plugin architecture to use the refactored commands.

**Comment by abbyachau** [comment]
Hi @cdlliuy thanks for creating this issue - very much appreciate it. We will be releasing this fix soon in CLI `v6.43`.  I'm closing this issue as fixed, please do not hesitate to get in touch if you have feedback. Thanks again.
